### PR TITLE
Smallfix

### PR DESCRIPTION
**Pull request by jonkiky**

### CODE REVIEW

**Review by iksheth — APPROVED** [review]

File: public/injectEnv.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ window.injectedEnv ={
 	REACT_APP_BACKEND_GETUSERINFO_API:'https://k9dc.essential-dev.com/fence/login/',
```
Comment (AjayDoddapaneni) [inline public/injectEnv.js:2]: what are the changes that made in package.json? 30000 line?

Comment (jonkiky) [reply public/injectEnv.js:2]: No change in package.json

Comment (AjayDoddapaneni) [reply public/injectEnv.js:2]: Sorry I mean package-lock?

**Comment by AjayDoddapaneni** [comment]
Lets see if the build works, merging PR

**Pull request merged**